src/main/java/com/cloudbees/jenkins/GitHubPushTrigger.java

```diff
@@ -336,6 +342,8 @@ public static boolean allowsHookUrlOverride() {
     /**
      * Set to false to prevent the user from overriding the hook URL.
      */
+    @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", 
+            justification = "Common Jenkins config pattern, which may be changed in the runtime")
     public static boolean ALLOW_HOOKURL_OVERRIDE = !Boolean.getBoolean(GitHubPushTrigger.class.getName() + ".disableOverride");
```

Comment:
groovy scripts can change private, and so this field could be marked as private.